clients/gtest/gemm_ex_gtest.cpp

```diff
@@ -709,7 +743,8 @@ INSTANTIATE_TEST_SUITE_P(quick_blas_batched_ex_small_hpa_half_single_out,
                                  ValuesIn(precision_hpa_half_single_out),
                                  ValuesIn(compute_hpa_half_single_out),
                                  ValuesIn(batch_count_range),
-                                 ValuesIn(is_fortran)));
+                                 ValuesIn(is_fortran),
+                                 ValuesIn(without_flags)));
```

Comment:
Catching up, could this one be with_and_without ?  And is strided now tested as looks like you completed the full implementation now, right @daineAMD ?  I'll leave this to you as you did more than me now

Reply:
I'm guessing that Tensile would support this precision config as well but not sure, I added it in another commit so we can see how it behaves.
Yes, strided_batched (along with batched) is tested, full implementation is complete so should be good to go, will wait for another CI run.